combat-trainer.lic

```diff
@@ -946,15 +946,15 @@ class TrainerProcess
 
   def initialize(settings)
     echo('New TrainerProcess') if $debug_mode_ct
-
+	@stun_skill = settings.stun_skill
```

Comment:
If this is only used on line 957, then it does not need to be an instance var. You could make it local to this method (`stun_skill`, no @), or inline `settings.stun_skill` on line 957. I think I'm in favor of the inlining.